src/main/resources/application.properties

```diff
@@ -23,12 +23,13 @@ ei.useSecureHttpBackend=false
 ei.backendInstancesPath=
 
 ###### EI Documentation Link Url ##########
-ei.eiffelDocumentationUrls={ "EI Frontend GitHub": "https://github.com/Ericsson/eiffel-intelligence-frontend",\
-                             "EI Backend GitHub": "https://github.com/Ericsson/eiffel-intelligence",\
+ei.eiffelDocumentationUrls={ "EI Frontend GitHub": "https://github.com/eiffel-community/eiffel-intelligence-frontend",\
+                             "EI Backend GitHub": "https://github.com/eiffel-community/eiffel-intelligence",\
                              "Eiffel Github": "https://github.com/Ericsson/eiffel",\
-                             "Eiffel Github main page": "http://ericsson.github.io/eiffel" }
+                             "Eiffel Github main page": "https://github.com/eiffel-community/eiffel",\
```

Comment:
Remove "https://github.com/Ericsson/eiffel" link from this list.

Reply:
link removed